<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerWidget</class>
 <widget class="QWidget" name="PlayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>139</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>139</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>139</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="contentWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#contentWidget {
border:1px solid #000000;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="artistLabel">
        <property name="styleSheet">
         <string notr="true">font-size:20px;
font-weight:bold;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="titleLabel">
        <property name="styleSheet">
         <string notr="true">font-size:18px;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>28</width>
          <height>2</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="playedLabel">
           <property name="styleSheet">
            <string notr="true">font-size:25px</string>
           </property>
           <property name="text">
            <string notr="true">00:00.0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="toPlayLabel">
           <property name="styleSheet">
            <string notr="true">font-size:25px</string>
           </property>
           <property name="text">
            <string notr="true">-00:00.0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QProgressBar" name="progressBar">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>15</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QProgressBar:horizontal {
border: 1px solid gray;
padding: 1px;
text-align: center;
}
QProgressBar::chunk:horizontal {
background: red;
}</string>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="textVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="playButton">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:0;</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/icons.qrc">
          <normaloff>:/playlist/icons/control_play_blue.png</normaloff>:/playlist/icons/control_play_blue.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="stopButton">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:0;</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/icons.qrc">
          <normaloff>:/playlist/icons/control_stop_blue.png</normaloff>:/playlist/icons/control_stop_blue.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QPushButton" name="numberButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>55</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>55</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:2px solid #000000;
border-radius:5px;
font-size:40px;
font-weight:bold;
background-color:#ffffff;</string>
        </property>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
